src/client/cl_main.c

```diff
@@ -535,6 +536,7 @@ static void Cl_InitLocal(void) {
 	// register our variables
 	cl_chat_sound = Cvar_Add("cl_chat_sound", "misc/chat", 0, NULL);
 	cl_draw_counters = Cvar_Add("cl_draw_counters", "1", CVAR_ARCHIVE, NULL);
+	cl_draw_position = Cvar_Add("cl_draw_position", "0", CVAR_ARCHIVE, NULL);
```

Comment:
I made it archive to fit the other cl_draw_* counters, but yeah I only used it to debug map issues so it can safely be made LO I suppose